test/parallel/test-child-process-spawnsync-validation-errors.js

```diff
@@ -101,7 +104,7 @@ if (!common.isWindows) {
   pass('shell', false);
   fail('shell', 0, invalidArgTypeError);
   fail('shell', 1, invalidArgTypeError);
-  fail('shell', [], invalidArgTypeError);
```

Comment:
```suggestion
  fail('shell', [], { code: 'ERR_INVALID_ARG_VALUE', name: 'TypeError' });
```

Using `common.expectsError` in combination with `assert.throws` is actually doing the same twice, thus this is a cleaner solution.